evalml/tests/automl_tests/test_automl.py

```diff
@@ -1078,6 +1081,9 @@ def test_catch_keyboard_interrupt(mock_fit, mock_score, mock_input,
     automl.search(X, y)
 
     assert len(automl._results['pipeline_results']) == number_results
+    if number_results != 5 and when_to_interrupt == 1:
```

Comment:
Why `!= 5` here?

Reply:
Only the first 2 `parametrize` args fail since they result in 0 pipelines being trained. I can simplify this though!